internal/appsec/waf_unit_test.go

```diff
@@ -0,0 +1,41 @@
+package appsec
```

Comment:
```suggestion
// Unless explicitly stated otherwise all files in this repository are licensed
// under the Apache License Version 2.0.
// This product includes software developed at Datadog (https://www.datadoghq.com/).
// Copyright 2016 Datadog, Inc.

//go:build appsec
// +build appsec

package appsec
```